sormas-base/dependencies/check-suppressions.xml

```diff
@@ -11,10 +11,30 @@
   </suppress>
   <suppress>
     <notes><![CDATA[
-    We are using newer Gradle version 5.4.1
+    Adresses older Payara Server
+    ]]></notes>
+    <cve>CVE-2022-37422</cve><!-- CVE is fixed with 5.2022.3 -->
```

Comment:
@StefanKock Why do we need suppressions for CVEs that are already fixed in the version of a dependency that we are using?

Reply:
Because it triggers on jars we have in the deploy path (launcher.jar) and on `fish.payara.security.connectors:security-connector-oidc-client:2x`